lib/modules/apostrophe-pager/public/css/user.less

```diff
@@ -29,3 +29,7 @@
   }
 
 }
+
+.apos-field-apply-to-subpages .apos-workflow-field-controls {
```

Comment:
Can we get a comment explaining this? It's not clear why we're removing something from view. If this is specific to sites using apos-workflow then this should be there, not in core.